Applications/Spire/Source/Charting/ChartView.cpp

```diff
@@ -13,6 +13,8 @@ using namespace Nexus;
 using namespace Spire;
 
 namespace {
+  const auto GAP_DIVISOR = 2.95;
```

Comment:
Make this local to the function using it.

Reply:
Is there no advantage to declaring it in an anonymous namespace?

Reply:
Use the most local scope as possible. If something is declared in an anonymous namespace, from my point of view it's a variable I have to be aware of and keep track of for the entire file. If it's local to a function, then I only need to be aware of it for that specific function, so on so forth.